integration/README.md

```diff
@@ -0,0 +1,118 @@
+```
+Verify that pods have been created.
+```sh
+$ kubectl get pods --namespace=kube-system
+```
+Log into the one of pods using the pod name returned by `kubectl get pods` command.
+```sh
+$ kubectl exec -ti snap-xxxxx bash --namespace=kube-system
+```
+Now that you are inside the pod with running Snap daemon, you may run command to list the plugins
+```sh
+$ snaptel plugin list
+``` 
+The output should be `No plugins found. Have you loaded a plugin?` as there are no plugins loaded yet. Let's download some...
+
+```sh
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-collector-docker/releases/download/5/snap-plugin-collector-docker_linux_x86_64" -o snap-plugin-collector-docker
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64" -o snap-plugin-publisher-file
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-heapster/releases/download/1/snap-plugin-publisher-heapster_linux_x86_64" -o snap-plugin-publisher-heapster
+```
+...and load them.
+```sh
+$ snaptel plugin load snap-plugin-collector-docker
+$ snaptel plugin load snap-plugin-publisher-file
+$ snaptel plugin load snap-plugin-publisher-heapster
+```
+Now, running the command
+```sh
+$ snaptel plugin list
+``` 
+will print information about three loaded plugins. To start the collection of metrics you can create task.
+```sh
+$ curl -sO https://raw.githubusercontent.com/intelsdi-x/kubesnap/integration/integration/tasks/docker-file.json
+$ snaptel task create -t ./tasks/docker-file.json
+``` 
+Command
+```sh
+$ snaptel task list
+``` 
+will provide information whether the metrics collection was successfull or not.
+```sh
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-collector-docker/releases/download/5/snap-plugin-collector-docker_linux_x86_64" -o snap-plugin-collector-docker
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64" -o snap-plugin-publisher-file
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-heapster/releases/download/1/snap-plugin-publisher-heapster_linux_x86_64" -o snap-plugin-publisher-heapster
+```
+and load them:
+```sh
+$ snaptel plugin load snap-plugin-collector-docker
+$ snaptel plugin load snap-plugin-publisher-file
+$ snaptel plugin load snap-plugin-publisher-heapster
+```
+The last step is to download task manifests and create them in order to start the collection of metrics.
+
+```sh
+$ curl -sO https://raw.githubusercontent.com/intelsdi-x/kubesnap/integration/integration/tasks/docker-file.json
+$ curl -sO https://raw.githubusercontent.com/intelsdi-x/kubesnap/integration/integration/tasks/docker-heapster.json
+$ snaptel task create -t ./tasks/docker-file.json
+$ snaptel task create -t ./tasks/docker-heapster.json
+```
+Use task watch to verify that tasks have been created correctly.
+```sh
+$ snaptel task watch <task_id>
+```
```

Comment:
Add subsections:
1. Using snap with heapster publisher (what is needed to use it)
2. Using snap in Tribe mode (indicate it's beta feature for now).
3. How to change configuration of snapteld when snap daemonset is running
4. Describe what changes have to be make in Dockerfile and snap.yaml when plugins have specific needs (i.e. Docker version, SW dependecies, K8s namespace, K8s mounts, etc.)